web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -712,6 +716,77 @@ export function DatabaseItem(props: SearchResultItem<SearchResultDatabase>) {
   );
 }
 
+export function AppItem(props: SearchResultItem<SearchResultApp>) {
+  const { searchResult } = props;
+  const app = searchResult.resource;
+
+  const $resourceFields = (
+    <ResourceFields>
+      {app.addrWithProtocol && (
```

Comment:
When looking at this code, the first question that pops in to my head is "When is `addrWithProtocol` empty?". Could we add a JSDoc with an answer? It's probably also worth redefining the `publicAddr` on the `tsh.App` type to add a JSDoc explaining when `publicAddr` is empty.

It'd be good to have some kind of documentation for each field in general, because at the moment it's not clear how `addrWithProtocol` differs from `publicAddr` and `endpointUri` and how `name` is different from `friendlyName`.

Reply:
Added!